src/DotNet/ModuleDefMD.cs

```diff
@@ -234,7 +234,20 @@ public sealed class ModuleDefMD : ModuleDefMD2, IInstructionOperandResolver {
 		public static ModuleDefMD Load(System.Reflection.Module mod, ModuleContext context, ImageLayout imageLayout) {
 			return Load(mod, new ModuleCreationOptions(context), imageLayout);
 		}
-
+		
+#if NETSTANDARD2_0
+		//TODO: works only on windows
+		[DllImport("Kernel32.dll", SetLastError = true, CharSet = CharSet.Unicode)]
+		static extern IntPtr GetModuleHandle(string lpModuleName);
+#endif
+		static IntPtr GetModuleHandle(System.Reflection.Module mod)
```

Comment:
`Marshal.GetHINSTANCE` exists on Windows and I think Mono also supports it. You could use reflection to get the method. If it doesn't exist, you could throw NotSupportedException

Reply:
Only do this if it's a netstandard 2.0 build of course.